### ISSUE TITLE
Missing notify heading label for dialogs

### ISSUE THREAD

**Issue by joe-replin** [adaptlearning/adapt-contrib-core#173]
## Bug Description:
Depending on the design, certain dialogues throughout an Adapt experience often do not have a meaningful label associated with them. For instance:
* Transcripts
* Question Feedback
* Hotgraphic pop-ups

## Steps to Reproduce:
Take away any particular notify title
On entry into a modal dialog, note the dialog title.

## Actual Behavior:
Dialog with aria-labelledby referencing an invalid id:
`<div role="dialog" aria-labelledby="notify-heading" aria-modal="true" class="notify is-visua11ysettings" style="visibility: visible;">`

## Expected Behavior:
My proposal is to at least include an invisible notify-heading able to be read by a screen reader that describes the purpose of the pop-up. And any designed title would override that default if added.

**Comment by oliverfoster** [comment]
Yup, lovely.

**Comment by chris-steele** [comment]
The invoker should provide the default heading. For example, in the case of question feedback this should be adapt-contrib-tutor: https://github.com/adaptlearning/adapt-contrib-tutor/issues/74
